sdk/common/src/main/java/io/opentelemetry/sdk/resources/Resource.java

```diff
@@ -52,9 +53,15 @@
   private static final String ERROR_MESSAGE_INVALID_VALUE =
       " should be a ASCII string with a length not exceed " + MAX_LENGTH + " characters.";
   private static final Resource EMPTY = create(Attributes.empty());
-
   private static final Resource TELEMETRY_SDK;
 
+  /**
+   * The MANDATORY Resource instance contains the mandatory attributes that must be used if they are
+   * not provided by the Resource that is given to an SDK signal provider.
+   */
+  public static final Resource MANDATORY =
```

Comment:
I don't think this needs to be separated from DEFAULT. Also, this would be the only `public` field in this class, the others are `private` and have getters.

Reply:
I like having it be a separate thing in the code, but agree it should be private, and can be accessed via the default Resource.

Reply:
done